game/client_types/bot.js

```diff
@@ -6,28 +6,30 @@
  * http://www.nodegame.org
  * ---
  */
-var ngc = require('nodegame-client');
-var J = ngc.JSUS;
+let ngc = require('nodegame-client');
+let J = ngc.JSUS;
 
 module.exports = function(treatmentName, settings, stager, setup, gameRoom) {
 
 
-    var channel = gameRoom.channel;
-    var logic = gameRoom.node;
+    let channel = gameRoom.channel;
+    let logic = gameRoom.node;
 
     stager.extendAllSteps(function(o) {
         o.cb = function() {
-            var node, stepObj, id;
-            stepObj = this.getCurrentStepObj();
-            id = stepObj.id;
-            node = this.node;
+            //var node, stepObj, id;
+            let stepObj = this.getCurrentStepObj();
+            let id = stepObj.id;
+            let node = this.node;
 
             // We do not actually play.
 
             if (id === 'decision') {
                 node.on('PLAYING', function() {
                     node.timer.random.exec(function() {
-                        var decision, departure;
+                        //var decision, departure;
+                        let decision = '';
+                        let departure = 0;
```

Comment:
This is correct, but it can be improved. No need to give any values here, just do:
`let decision, departure;`